.github/rfc-format/check-rfc-format.py

```diff
@@ -241,6 +241,10 @@ def main():
         if not os.path.exists(readme_file_path):
             continue
 
+        # The RFC template has special formatting, so ignore it in the check
+        if rfc_dir.endswith("template"):
```

Comment:
Fine, but let's do this instead at the beginning of the loop so we bail out early

```
if entry_name == 'template':
    continue
```